README.md

```diff
@@ -11,7 +11,7 @@ Kolibri Studio is a web application designed to deliver educational materials to
 - Creating learning pathways and assessments
 - Uploading new content through the web interface or programatically using [ricecooker-powered](https://github.com/learningequality/ricecooker) content import scripts
 
-Kolibri Studio uses [Django](https://www.djangoproject.com/) for the backend and is transitioning from [Backbone.js](https://backbonejs.org/) to [Vue.js](https://vuejs.org/) for the frontend.
+Kolibri Studio uses [Django](https://www.djangoproject.com/) for the backend and is transitioning from [Backbone.js](https://backbonejs.org/) to [Vue.js](https://vuejs.org/) for the frontend. If you are looking for help installing, configuring and using Kolibri Studio, please refer to the [User Guide](https://kolibri-studio.readthedocs.io/en/latest/).
```

Comment:
Thank you for the contribution @priyanka-choubey!

Note that Kolibri Studio is not a product that we make available for users to install locally. For that reason, Studio user guide does not cover the installation or configuration, but just the general instructions for making custom content channels, uploading and organizing resources.

Reply:
@radinamatic Thanks for the review. Since I am new to this repo, I was not aware about the studio user guide. I have modified the string to "If you are looking for help setting up custom content channels, uploading and organizing resources using Kolibri Studio, please refer to the User Guide". Can you kindly review again.

Reply:
That sounds better, and I would only suggest separating the phrase into a new paragraph.

I´m also thinking that it wouldn't be bad to have some links for the Studio developer docs here, instead of just a pointer for the user docs. @jayoshih @ivanistheone What would you suggest adding here?